schlib/schlib.py

```diff
@@ -36,7 +36,7 @@ def __init__(self, filename, create = False):
 
         else:
             if not os.path.isfile(self.filename):
-                sys.stderr.write('"{filename}" is not a valid .dcm file\n'.format(filename=self.filename))
+                sys.stderr.write('"{filename}" does not exist'.format(filename=self.filename))
```

Comment:
please add a linebreak `\n` at the end of this error message, as currently the output is e.g.:

```
"d:\KiCAD\kicad-library\library\Altera.dcm" does not existchecking component: EP1210
  Violating Rule 3.8
    Description and keywords properties contains information about the component.
```